lib/ansible/module_utils/facts/system/service_mgr.py

```diff
@@ -98,6 +98,9 @@ def collect(self, module=None, collected_facts=None):
         if proc_1 == "COMMAND\n":
             proc_1 = None
 
+        if proc_1 is None:
+            proc_1 = os.readlink('/sbin/init')
```

Comment:
this will produce OSError when not a symlink, so we either want to test it is a symlink first or catch the exception

Reply:
Good catch. Fixed.